query/parse.go

```diff
@@ -403,6 +424,8 @@ var prefixes = map[string]int{
 	"content:":  tokContent,
 	"f:":        tokFile,
 	"file:":     tokFile,
+	"fork:":     tokFork,
```

Comment:
Bikeshedding on `fork:` welcome.

I picked `fork:` because it matches the naming scheme of the zoekt gitconfig setting, but on the other hand we use `archived:` not `archive:`.

On the other other hand, `forked:` might be ambiguous: does "forked" indicate that it is forked from another project or that it has been forked by another project?  Maybe `fork:` is the right call 🌞.

Reply:
I think `fork:` is good.

> but on the other hand we use archived: not archive:.

can you elaborate? Not sure I understand.